pages/windows/forfiles.md

```diff
@@ -0,0 +1,27 @@
+# forfiles
+
+> Select one or more files to execute a specified command on.
+
+- Search for files in the current directory:
+
+`forfiles`
+
+- Search for files in a specific directory:
+
+`forfiles /p {{path/to/directory}}`
+
+- Run the specified command for each file:
+
+`forfiles /c "{{command}}"`
+
+- Search for files using a specific glob mask:
+
+`forfiles /m {{glob_pattern}}`
+
+- Search for files recursively:
+
+`forfiles /s`
+
+- Search for files with a last-modified date greater than 5 days old:
```

Comment:
Maybe simpler to say - "Search for files older than 5 days", to keep parity with the `find` page.